src/sections/footer.html

```diff
@@ -16,17 +15,15 @@
                 <p class="text-white md:block hidden">stetig wachsender Markt und Talentpool</p>
             </div>
             <div class="flex justify-center gap-2 items-center">
-                <svg  width="70" height="40" viewBox="0 0 74 46" fill="none"
-                    xmlns="http://www.w3.org/2000/svg">
+                <svg width="70" height="40" viewBox="0 0 74 46" fill="none" xmlns="http://www.w3.org/2000/svg">
                     <path
                         d="M63.1179 9.88708C59.6921 6.74169 55.6205 4.24867 51.1379 2.55176C46.6552 0.854861 41.8502 -0.012351 36.9999 0.000132888H36.8678C16.541 0.0636853 0 15.4343 0 34.2367V41.1579C0 42.4421 0.556885 43.6737 1.54815 44.5818C2.53941 45.4898 3.88385 46 5.2857 46H68.7142C70.116 46 71.4604 45.4898 72.4517 44.5818C73.443 43.6737 73.9999 42.4421 73.9999 41.1579V33.8948C74.0134 29.4321 73.0584 25.0115 71.1902 20.89C69.3221 16.7685 66.5781 13.0284 63.1179 9.88708ZM68.7142 41.1579H34.2613L52.3516 18.3698C52.7643 17.8505 52.9348 17.2023 52.8258 16.5678C52.7168 15.9333 52.337 15.3644 51.7702 14.9864C51.2033 14.6084 50.4957 14.4521 49.8031 14.552C49.1104 14.6519 48.4895 14.9997 48.0768 15.519L27.7235 41.1579H5.2857V34.2367C5.2857 33.3046 5.33526 32.3846 5.42776 31.4737H13.2143C13.9152 31.4737 14.5874 31.2187 15.083 30.7646C15.5787 30.3106 15.8571 29.6948 15.8571 29.0527C15.8571 28.4106 15.5787 27.7948 15.083 27.3407C14.5874 26.8867 13.9152 26.6316 13.2143 26.6316H6.36597C9.77525 14.8714 20.9017 5.98011 34.3571 4.94512V12.1054C34.3571 12.7475 34.6355 13.3633 35.1311 13.8173C35.6268 14.2713 36.299 14.5264 36.9999 14.5264C37.7009 14.5264 38.3731 14.2713 38.8687 13.8173C39.3643 13.3633 39.6428 12.7475 39.6428 12.1054V4.94209C46.1992 5.44755 52.4225 7.80588 57.4526 11.6912C62.4828 15.5766 66.0716 20.7972 67.7231 26.6316H60.7856C60.0847 26.6316 59.4124 26.8867 58.9168 27.3407C58.4212 27.7948 58.1427 28.4106 58.1427 29.0527C58.1427 29.6948 58.4212 30.3106 58.9168 30.7646C59.4124 31.2187 60.0847 31.4737 60.7856 31.4737H68.6051C68.6745 32.2757 68.7142 33.0807 68.7142 33.8948V41.1579Z"
                         fill="white" fill-opacity="0.7" />
                 </svg>
                 <p class="text-white md:flex hidden">sehr gutes Investitionsklima</p>
```

Comment:
And also here add the correct font family

Reply:
done